shark_turbine/kernel/lang/wave_types.py

```diff
@@ -109,3 +110,11 @@ def __class_getitem__(
             symbolic_shape=shape,
             dtype=dtype,
         )
+
+
+class IndexMapping:
+
+    mapping_func: ClassVar[Callable[..., Any]]
```

Comment:
Won't this always be a mapping from a tuple of IndexExpr to a tuple of IndexExpr?

Reply:
Yeah, will update